tools/changelog/src/update-version-histories.js

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs';
+
+const packagesDir = '../../packages';
+
+function getAdaptorsFromDir() {
+  return fs
+    .readdirSync(packagesDir, { withFileTypes: true })
+    .filter(dirent => dirent.isDirectory())
+    .map(dirent => dirent.name);
+}
+
+async function fetchVersions(adaptor) {
```

Comment:
Can we add some logging here, like in an adaptor code?

Just to give the user some feedback.

So each time you fetch versions, you should log "fetching common" or whatever, so we can see the traffic